### DISCUSSION TITLE
左下角资源更新弹窗，真的有在更新什么资源么

### DISCUSSION THREAD

**Discussion by Private233** [MaaAssistantArknights/MaaAssistantArknights#7200] [General]
<img width="600" alt="1C0E88C978E390CCE8095415C90D5E9C" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/108540840/a3aec2de-7653-4afa-ab3c-cb7e127976b0">
<img width="930" alt="{A1FF2547-BB56-497e-94E7-416EC5BBFA96}" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/108540840/0f8a4169-74d6-4f42-ae75-0f364442591c">
每次打开右下角都会弹通知，但也不知道在更新什么，进度如何。
另外注意到开新活动有新内容的时候窗口右上角也有弹小通知。
所以右下角的通知不知道是什么意义

**Comment by ABA2396** [comment]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/99072975/7e0490f4-a8a7-41a0-bae8-825aa01794e7)

游戏资源更新的话就是公招数据，地图数据之类的东西

**Reply by Private233** [reply]
我每次开都弹，真的是获取更新这么频繁吗？也不知道进度。不发系统通知而做到ui内会不会好些
